stat.go

```diff
@@ -648,6 +648,71 @@ func KullbackLeibler(p, q []float64) float64 {
 	return kl
 }
 
+// OrdinaryLinearRegression computes the simple linear regression of x and y.
```

Comment:
If there were no weights parameter, I would be happy to do that, but the presence of the weights parameter leads to the possible ambiguity that this does ordinary and weighted least squares linear regression. It doesn't AFAIU.